charts/federation-v2/charts/controllermanager/templates/clusterrole.yaml

```diff
@@ -13,35 +13,57 @@ rules:
   resources:
   - '*'
   verbs:
-  - '*'
+  - get
+  - watch
+  - list
+  - update
 - apiGroups:
   - multiclusterdns.federation.k8s.io
   resources:
   - '*'
   verbs:
-  - '*'
+  - get
+  - watch
+  - list
+  - create
+  - update
+  - delete
 - apiGroups:
   - core.federation.k8s.io
   resources:
   - '*'
   verbs:
-  - '*'
+  - get
+  - watch
+  - list
+  - create
+  - update
 - apiGroups:
   - types.federation.k8s.io
   resources:
   - '*'
   verbs:
-  - '*'
+  - get
+  - watch
+  - list
+  - update
 - apiGroups:
   - clusterregistry.k8s.io
   resources:
-  - '*'
+  - clusters
   verbs:
-  - '*'
+  - get
 - apiGroups:
   - ""
   resources:
-  - '*'
+  - configmaps
```

Comment:
What does federation use configmaps for?

I'm thinking that federation should only have access to secrets in the federation system namespace.  I'm assuming that would require a role rather than a cluster role?

Reply:
it is leader elector which uses `configmap`